deploy/olm-catalog/ibm-commonui-operator/1.2.0/ibm-commonui-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -69,7 +69,7 @@ metadata:
     repository: https://github.com/IBM/ibm-commonui-operator
     support: IBM
   name: ibm-commonui-operator.v1.2.0
-  namespace: placeholder
+  namespace: ibm-common-services
```

Comment:
For line 67: description:

Can this be changed to:  
```
This operator delivers the common header API and the identity and access pages for IBM Cloud Platform Common Services.
```

or, if space permits: 

```
The IBM Common Web UI delivers the common header API and the identity and access pages for IBM Cloud Platform Common Services.
```